addons/website/static/src/core/website_edit_service.js

```diff
@@ -94,13 +94,22 @@ PublicRoot.include({
 
 // Patch Colibri.
 
+const HISTORY_SUFFIX_RE = /\.keepInHistory$/;
+
```

Comment:
We agree you don't need this regex anymore ? :slightly_smiling_face: 
EDIT: this is what makes the enterprise build red so you have remove it :+1: